pulp_container/app/registry_api.py

```diff
@@ -738,15 +701,15 @@ def put(self, request, path, pk=None):
                 raise Exception("The digest did not match")
             try:
                 artifact = Artifact.init_and_validate(uploaded_file)
-                artifact.save()
+                artifact.touch()
```

Comment:
I think the original way was correct. (This is creating a new artifact, no? Because if it already exists, we go down the ecxept path where touching is actually desired.)

Reply:
So, are not we saving the object either way in this case?

https://github.com/pulp/pulpcore/blob/ad319b1963fd7462bbf9e5562ead263a42481db8/pulpcore/app/models/content.py#L389

Reply:
touch is limited to the timestamp field. I'm not sure how this behaves with new objects, but at least touch has a different intention that save.

Reply:
Okay, thanks for the clarification. I will change it back.